### PR TITLE
Add Docker Auto Push script

### LINKED ISSUE #502: Create Documentation explaining how exported Images can be pushed

### LINKED ISSUE #503: Create Simple Deploy Script

### PR DESCRIPTION
**Pull request by c-mueller**
This PR Resolves:
 - Resolves #503 
 - Resolves #502 (Indirectly)

### CODE REVIEW

**Comment by c-mueller** [comment]
I Consider this done. it therefore is ready to review

File: server/src/main/resources/kubernetes/docker/push-images.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+read reg_url
+
+# Check if the string is not empty
+if [ -z "$reg_url" ]; then
+  # Add Trailing slash if it does not exist
+  if [ $reg_url != */ ]
+  then
+    reg_url="$reg_url/"
+  fi
+fi
```
Comment (hnicke) [inline server/src/main/resources/kubernetes/docker/push-images.sh]: [ -z "$reg_url" ] && [[ "$reg_url" =~ .*/$ ]] && reg_url="$reg_url/"

File: server/src/main/resources/kubernetes/docker/push-images.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  exit 1
+fi
+
+echo "Creating Backup of Kubernetes Resource"
+cp kubernetes-resources/complete.yml kubernetes-resources/complete-original.yml
+
+cd docker
+for file in *.tar.gz
+do
+  echo "Loading image archive: $i"
```
Comment (hnicke) [inline server/src/main/resources/kubernetes/docker/push-images.sh]: $i => $file

File: server/src/main/resources/kubernetes/docker/push-images.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  #Import the image with pv (or cat if pv isnt installed)
+  if ! [ -x "$(command -v "pv")" ]; then
+    cat $file | docker load
```
Comment (hnicke) [inline server/src/main/resources/kubernetes/docker/push-images.sh]: docker load < $file

**Pull request merged**